oxli/partition.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
```

Comment:
(Explanation: if you're fixing syntax and deleting things, fine; but this kind of bigger rearrangement needs to be thought through separately from all the other changes.)